src/include/common/constants.h

```diff
@@ -199,6 +204,17 @@ struct ImportDBConstants {
     static constexpr char COPY_NAME[] = "copy.cypher";
 };
 
+struct WarningConstants {
+    static constexpr uint64_t WARNING_TABLE_NUM_COLUMNS = 5;
+    static constexpr std::array WARNING_TABLE_COLUMN_NAMES{"query_id", "message", "file_path",
```

Comment:
You can use static array like:
```
 static constexpr const char* BOOL_CSV_PARSING_OPTIONS[] = {"HEADER", "PARALLEL",
        "IGNORE_ERRORS"};
```

Reply:
an `std::array` is just a wrapper around a static array with some convenience methods that I use such as `size()` and iterators